iAM-legacy/RoadCareSecurityOperations/SecOps.cs

```diff
@@ -74,7 +74,10 @@ public bool ReadOnly
 
 		private AccessLevel GetPermissions( Dictionary<String, String> actionSpecifier )
 		{
-			AccessLevel grantedLevel = AccessLevel.Default;
+            return AccessLevel.CreateDestroy;
```

Comment:
Why was this return added here? The rest of the function now does nothing. What is the purpose of the change to this function?